### ISSUE TITLE
Several Errors Encountered at the Same Time

### ISSUE THREAD

**Issue by jglenn1562** [paradoxReborn/SurvivalReborn-Spacewalk#4]
Encountered several issues shortly into a new, offline game. There may have been a conflict with the "Populated Worlds Ai Enabled NPCs" mod, as I got the minor lag spike that's generally caused by NPC's spawning in right when the errors occurred. All I did was drop in with a spawn pod, mine some ice, and began running to where stone is at before the errors.
Log: [SpaceEngineers_20220827_151044214.log](https://github.com/paradoxReborn/SurvivalReborn-Spacewalk/files/9438706/SpaceEngineers_20220827_151044214.log)

<details>
  <summary>Mod List</summary>

  ```js
  <Mods>
    <ModItem FriendlyName="ACS Assertive Bunkers">
      <Name>2473174197.sbm</Name>
      <PublishedFileId>2473174197</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Aerodynamic Physics">
      <Name>571920453.sbm</Name>
      <PublishedFileId>571920453</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="AiEnabled [Beta]">
      <Name>2596208372.sbm</Name>
      <PublishedFileId>2596208372</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="AQD - Emissive Colors">
      <Name>2711430394.sbm</Name>
      <PublishedFileId>2711430394</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Aryx Weapon Enterprises [WeaponCore]">
      <Name>2530716039.sbm</Name>
      <PublishedFileId>2530716039</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Assertive Combat Systems">
      <Name>1902970975.sbm</Name>
      <PublishedFileId>1902970975</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Autofill Bottles">
      <Name>2308085590.sbm</Name>
      <PublishedFileId>2308085590</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Automatic Ore Pickup">
      <Name>657749341.sbm</Name>
      <PublishedFileId>657749341</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Bot_spawner">
      <Name>2336089504.sbm</Name>
      <PublishedFileId>2336089504</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Build Vision 3.0">
      <Name>1697184408.sbm</Name>
      <PublishedFileId>1697184408</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="BuildInfo (extra block+terminal info, air leak finder)">
      <Name>514062285.sbm</Name>
      <PublishedFileId>514062285</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Concrete Tool - placing voxels in survival">
      <Name>396679430.sbm</Name>
      <PublishedFileId>396679430</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Crew Enabled">
      <Name>2803081060.sbm</Name>
      <PublishedFileId>2803081060</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Defense Shields - v2.0(38)">
      <Name>1365616918.sbm</Name>
      <PublishedFileId>1365616918</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Definition Extension API">
      <Name>2756894170.sbm</Name>
      <PublishedFileId>2756894170</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Dune 2021 Jukebox Music">
      <Name>Dune 2021 Jukebox Music</Name>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Elite Dangerous Style Jump Effect">
      <Name>2752881305.sbm</Name>
      <PublishedFileId>2752881305</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Film &amp; Game Music">
      <Name>874485234.sbm</Name>
      <PublishedFileId>874485234</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Hostile Takeover">
      <Name>2694751775.sbm</Name>
      <PublishedFileId>2694751775</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Imber Corporation (Formerly Air Traffic)">
      <Name>973526550.sbm</Name>
      <PublishedFileId>973526550</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Improved Atmosphere Visuals">
      <Name>2794826879.sbm</Name>
      <PublishedFileId>2794826879</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Independent Contractors (Formerly Reddit Custom Encounters)">
      <Name>445996030.sbm</Name>
      <PublishedFileId>445996030</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Infestation Enabled">
      <Name>2809500674.sbm</Name>
      <PublishedFileId>2809500674</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Modular Encounters Systems">
      <Name>1521905890.sbm</Name>
      <PublishedFileId>1521905890</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="NPC Defense Shields Provider">
      <Name>2043339470.sbm</Name>
      <PublishedFileId>2043339470</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="NPC Weapon Upgrades">
      <Name>1555044803.sbm</Name>
      <PublishedFileId>1555044803</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Paint Gun - realistic painting for survival + special creative features">
      <Name>500818376.sbm</Name>
      <PublishedFileId>500818376</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Parallax Concepts (Formerly Surface Occupation)">
      <Name>1135484957.sbm</Name>
      <PublishedFileId>1135484957</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="PassageIntersections">
      <Name>946724937.sbm</Name>
      <PublishedFileId>946724937</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Planet Rotate [Beta 0.5]">
      <Name>2811518277.sbm</Name>
      <PublishedFileId>2811518277</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Populated Worlds Ai Enabled NPCs">
      <Name>2621488886.sbm</Name>
      <PublishedFileId>2621488886</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Pravda Heavy Industries">
      <Name>2852753555.sbm</Name>
      <PublishedFileId>2852753555</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Projections To Assembler">
      <Name>1289485324.sbm</Name>
      <PublishedFileId>1289485324</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Realistic Thrusters - offset forces for thrusters (disabled for NPCs!)">
      <Name>575893643.sbm</Name>
      <PublishedFileId>575893643</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Relative Top Speed">
      <Name>1359618037.sbm</Name>
      <PublishedFileId>1359618037</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Rotary Airlock">
      <Name>1359954841.sbm</Name>
      <PublishedFileId>1359954841</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Rotate skybox with the Sun">
      <Name>2830474647.sbm</Name>
      <PublishedFileId>2830474647</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Screen Gunk Be Gone!">
      <Name>1289657379.sbm</Name>
      <PublishedFileId>1289657379</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Space Just Got Real (1.188 Update 4)">
      <Name>1136188499.sbm</Name>
      <PublishedFileId>1136188499</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Survival Reborn: Spacewalk">
      <Name>2836145064.sbm</Name>
      <PublishedFileId>2836145064</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Text HUD API">
      <Name>758597413.sbm</Name>
      <PublishedFileId>758597413</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Walkable Conveyor System">
      <Name>2840158829.sbm</Name>
      <PublishedFileId>2840158829</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="WeaponCore - 2.2(20)-Final">
      <Name>1918681825.sbm</Name>
      <PublishedFileId>1918681825</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="WeaponCore - Replace Vanilla Weapons">
      <Name>1931509062.sbm</Name>
      <PublishedFileId>1931509062</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Whip's Camera Overlay">
      <Name>704012607.sbm</Name>
      <PublishedFileId>704012607</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Zardos Connector Passageway &amp; More">
      <Name>2783691448.sbm</Name>
      <PublishedFileId>2783691448</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
    </ModItem>
    <ModItem FriendlyName="Camera Panning - rotate camera view (+reset 1st person character view)">
      <Name>806331071.sbm</Name>
      <PublishedFileId>806331071</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
    <ModItem FriendlyName="Small Spider">
      <Name>2178783350.sbm</Name>
      <PublishedFileId>2178783350</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
    <ModItem FriendlyName="Rich HUD Master">
      <Name>1965654081.sbm</Name>
      <PublishedFileId>1965654081</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
    <ModItem FriendlyName="Assertive Loot">
      <Name>2391318591.sbm</Name>
      <PublishedFileId>2391318591</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
    <ModItem FriendlyName="In Depth LCDs">
      <Name>2839553181.sbm</Name>
      <PublishedFileId>2839553181</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
    <ModItem FriendlyName="Shower Light">
      <Name>2841920953.sbm</Name>
      <PublishedFileId>2841920953</PublishedFileId>
      <PublishedServiceName>Steam</PublishedServiceName>
      <IsDependency>true</IsDependency>
    </ModItem>
  </Mods>
  ```
</details>

**Comment by paradoxReborn** [comment]
The mod has been updated to account for characters that do not have an oxygen component, and it will not produce these errors anymore. Please update your mod.

Did you notice anything strange other than the error messages?

**Comment by jglenn1562** [comment]
> The mod has been updated to account for characters that do not have an oxygen component, and it will not produce these errors anymore. Please update your mod.

Great, thank you! I have updated the mod.

> Did you notice anything strange other than the error messages?

The only other issue I noticed was that it seemed that gravity felt like I was on the moon. I was on the snow planet (Triton?), and its gravity is supposed to match Earthlike's gravity. I'm unsure whether this was related to this mod, my sim speed, or another mod, I can take a look at the sim speed later.
